.github/workflows/cache.yml

```diff
@@ -0,0 +1,49 @@
+on:
+  workflow_call:
+    inputs:
+      minimal:
+        description: minimal dataset
+        type: boolean
+        required: false
+        default: true
+    outputs:
+      version:
+        value: ${{ jobs.download.outputs.version }}
+      cache_path:
+        value: ${{ jobs.download.outputs.cache_path }}
+      cache_key:
+        value: ${{ jobs.download.outputs.cache_key }}
+  workflow_dispatch:
+    inputs:
+      minimal:
+        description: minimal dataset
+        type: boolean
+        required: false
+        default: true
+
+env:
+  DATA_URL: https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz
+  MINIMAL_DATA_URL: https://stsci.box.com/shared/static/gkbnn3jwcootr9riwv0rv1t8rpvxl3vr.gz
```

Comment:
This is nice, to be able to toggle between either the smaller or full datasets based on an input parameter.  Much appreciated for this nice lesson in how to set up a Github workflow with an input switch, which I definitely did not know how to do. :-) 

BTW, just for context it _used_ to be the case that the JWST zip file was much larger, several hundred MB, so there was a more substantial difference between full and minimal. After various revisions in what's included in the dataset, this is no longer the case, so there's less need for having two versions of the data files. We may eventually simplify to just the full data only, but haven't yet, so this is great in any case. Just letting you know this potential future change as an FYI.